src/main.rs

```diff
@@ -88,10 +88,12 @@ fn main() -> Result<()> {
             let flag_value = match arg.as_ref() {
                 "--commands" | "-c" => {
                     // FIXME: Use proper quoting. `escape_quote_string()` can't be used for now due to https://github.com/nushell/nushell/issues/5383.
-
                     args.next().map(|a| format!("`{}`", a))
                 }
-                "--config" | "--env-config" => args.next().map(|a| escape_quote_string(&a)),
+                "--config" | "--env-config" => {
+                    // FIXME: Use proper quoting. `escape_quote_string()` can't be used because, as Windows path, foo\bar.nu and "foo\\bar.nu" are not equal.
+                    args.next().map(|a| format!("`{}`", a))
+                }
```

Comment:
`foo\bar.nu` and `"foo\\bar.nu"` _should_ be equal, actually.

Reply:
it would be nice to find out why they're not equal. That seems like it could be the real fix.

Reply:
For the simple case, the seem to be the same:

```
> let-env FOO = "foo\\bar.nu"
> $env.FOO
foo\bar.nu
> let-env FOO = `foo\bar.nu`
> $env.FOO
foo\bar.nu

Reply:
I suspect the path argument is mistakenly parsed with `nu_parser::trim_quotes()` where `nu_parser::unescape_unquote_string()` should be used.

Reply:
Probably this:

https://github.com/nushell/nushell/blob/80d57d70cd48fa0fa40181eaa77fcddea4b3af75/crates/nu-parser/src/parser.rs#L1953-L1958

There seem to be a dozen of mistakes of this sort, though.

Reply:
I'd be happy to fix them